No "com.ibm.websphere.monitor.jmx.Counter" when calling jax-rs endpoint.

It is impossible to call rest endpoint when using OpenLiberty 20.0.0.7 or higher inside docker container with included feature `microProfile-3.3`.

When call to any rest endpoint I get the following exception and HTTP response code 500:

```
[WARNING ] CWMOT0009W: The OpenTracing exception mapper detected and is handling an unhandled exception from the JAX-RS application. 
                                                                                                               java.lang.NoClassDefFoundError: com.ibm.websphere.monitor.jmx.Counter
        at java.base/java.lang.Class.getDeclaredMethods(Class.java:1130)
        at java.base/java.lang.Class.getMethodSet(Class.java:1758)
        at java.base/java.lang.Class.addInterfaceMethods(Class.java:1808)
        at java.base/java.lang.Class.getMethodSet(Class.java:1767)
        at java.base/java.lang.Class.getMethods(Class.java:1684)
        at java.management/com.sun.jmx.mbeanserver.MBeanIntrospector.getMethods(MBeanIntrospector.java:181)
        at java.management/com.sun.jmx.mbeanserver.MBeanAnalyzer.initMaps(MBeanAnalyzer.java:128)
        at java.management/com.sun.jmx.mbeanserver.MBeanAnalyzer.<init>(MBeanAnalyzer.java:118)
        at java.management/com.sun.jmx.mbeanserver.MBeanAnalyzer.analyzer(MBeanAnalyzer.java:102)
        at java.management/com.sun.jmx.mbeanserver.MXBeanIntrospector.getAnalyzer(MXBeanIntrospector.java:74)
        at java.management/com.sun.jmx.mbeanserver.MBeanIntrospector.getPerInterface(MBeanIntrospector.java:192)
        at java.management/com.sun.jmx.mbeanserver.MBeanSupport.<init>(MBeanSupport.java:138)
        at java.management/com.sun.jmx.mbeanserver.MXBeanSupport.<init>(MXBeanSupport.java:66)
        at java.management/com.sun.jmx.mbeanserver.Introspector.makeDynamicMBean(Introspector.java:203)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerObject(DefaultMBeanServerInterceptor.java:888)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerMBean(DefaultMBeanServerInterceptor.java:320)
        at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.registerMBean(JmxMBeanServer.java:522)
        at com.ibm.ws.kernel.boot.jmx.service.MBeanServerForwarderDelegate.registerMBean(MBeanServerForwarderDelegate.java:221)
        at [internal classes]
Caused by: java.lang.ClassNotFoundException: com.ibm.websphere.monitor.jmx.Counter cannot be found by com.ibm.websphere.appserver.api.jaxrs20_1.0.47.cl201220201111-0736
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:516)
        ... 1 more

[WARNING ] Interceptor for {http://boundary.junk.srnjak.com/}MyResource has thrown exception, unwinding now
com.ibm.websphere.monitor.jmx.Counter
```
It is interesting that the problem is present when I use docker container, only (Using OpenLiberty maven plugin, everything works fine).

I tried to use docker images (all causes same error):
```
openliberty/open-liberty:20.0.0.7-full-java11-openj9-ubi
openliberty/open-liberty:20.0.0.8-full-java11-openj9-ubi
openliberty/open-liberty:20.0.0.9-full-java11-openj9-ubi
openliberty/open-liberty:20.0.0.10-full-java11-openj9-ubi
openliberty/open-liberty:20.0.0.11-full-java11-openj9-ubi
openliberty/open-liberty:20.0.0.12-full-java11-openj9-ubi
```

The last image where it works is:
```
openliberty/open-liberty:20.0.0.6-full-java11-openj9-ubi
```

Additional investigation:
- It seems, that there is problem with OSGI bundles.
- It works if I use `microProfile-3.2` OpenLiberty feature.
- It works if instead of `microProfile-3.3` there are included specific features:
```
json-1.0
jwt-1.0
mpConfig-1.4
mpFaultTolerance-2.1
mpHealth-2.2
mpJwt-1.1
mpOpenAPI-1.1
mpOpenTracing-1.3
mpRestClient-1.4
opentracing-1.3
```
Instead of `monitor-1.0` and `mpMetrics-2.3` I had to include `mpMetrics-2.2`.

I made POC project on github for this problem (branch docker-liberty):
https://github.com/srnjak/rest-poc/tree/docker-liberty

--- a/dev/com.ibm.websphere.appserver.api.jaxrs20/bnd.bnd
+++ b/dev/com.ibm.websphere.appserver.api.jaxrs20/bnd.bnd
@@ -18,8 +18,8 @@ Bundle-SymbolicName: com.ibm.websphere.appserver.api.jaxrs20
 Import-Package: \
   com.ibm.websphere.jaxrs.providers.json4j, \
   com.ibm.websphere.jaxrs20.multipart, \
-  com.ibm.websphere.jaxrs.monitor;resolution:=optional, \
-  com.ibm.websphere.monitor.jmx;resolution:=optional, \
+  com.ibm.websphere.jaxrs.monitor;resolution:=dynamic, \
+  com.ibm.websphere.monitor.jmx;resolution:=dynamic, \
   com.ibm.websphere.ras, \
   com.ibm.ws.jaxrs20.multipart.impl;resolution:=optional, \
   javax.activation, \

